draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -3829,167 +3913,354 @@ Server: example-server/0.1
 }
          ]]></artwork>
           </figure>
-        </section>
+    </section>
+
+  </section>
+  <section title="Changing, Cancelling and Deleting Triggers" anchor="examples-changing-triggers">
+    <section title="Modifying Triggers" anchor="examples-modifying-trigger">
+    <t>
+    uCDN can modify triggers while they are in "pending" state, for example adjust their specs and label attributes.
```

Comment:
Maybe: uCDN can modify triggers while they are in the "pending" state.  One example of this might be to adjust a trigger's "specs" and "labels" attributes.